### ISSUE TITLE
ns-eval, ns-render, ns-export load the whole training data (including images and transforms.json)

### ISSUE THREAD

**Issue by f-dy** [nerfstudio-project/nerfstudio#2050]
**Describe the bug**
The three commands `ns-eval`, `ns-render`, `ns-export` should not need the training data:
- `ns-eval` only need to load the test data (10% of the dataset by default)
- `ns-render` and `ns-export` do not need any training data

This is inefficient, especially when rendering or exporting from a pre-trained model on a different machine, where one may not want to transfer all training data with the checkpoint.

**To Reproduce**
any output of these three commands starts with:
```
Setting up training dataset...
Caching all 270 images.
```

**Expected behavior**
Only load data that is required :
- for ns-eval, transforms.jso + the test set
- for ns-render and ns-export, no transforms.json nor images

**Comment by tancik** [comment]
I'm a little confused how the provided changes solves the issue?

**Comment by f-dy** [comment]
commented in the PR. What confused me was why images were required at inference time :-)
(and btw huge congrats!)
